keyboards/handwired/xealous/keymaps/default/keymap.c

```diff
@@ -0,0 +1,105 @@
+#include QMK_KEYBOARD_H
+#include "config.h"
```

Comment:
This isn't needed.  It's included by the make system automatically

Reply:
Which ones?
When i remove both, it doesn't build

Reply:
It builds when i remove config.h, but not if i remove 
```
#include QMK_KEYBOARD_H
```

Reply:
Yeah, you need the keyboard on there.  The config.h is actually automatically included by the make script, IIRC.  So you don't need that included.